core/src/main/java/org/moditect/commands/GenerateModuleInfo.java

```diff
@@ -187,78 +187,78 @@ private void updateModuleInfo(Map<String, Boolean> optionalityPerModule, ModuleD
             moduleDeclaration.setOpen( true );
         }
 
-        List<ModuleRequiresStmt> requiresStatements = moduleDeclaration.getNodesByType( ModuleRequiresStmt.class );
+        List<ModuleRequiresDirective> requiresStatements = moduleDeclaration.findAll( ModuleRequiresDirective.class );
 
-        for ( ModuleRequiresStmt moduleRequiresStmt : requiresStatements ) {
-            if ( Boolean.TRUE.equals( optionalityPerModule.get( moduleRequiresStmt.getNameAsString() ) ) ) {
-                moduleRequiresStmt.addModifier( Modifier.STATIC );
+        for ( ModuleRequiresDirective moduleRequiresDirective : requiresStatements ) {
+            if ( Boolean.TRUE.equals( optionalityPerModule.get( moduleRequiresDirective.getNameAsString() ) ) ) {
+                moduleRequiresDirective.addModifier( Modifier.Keyword.STATIC );
             }
```

Comment:
Modifiers used to be enums, now they are true nodes. No more exceptional status for them. To indicate what kind of modifier a Modifier is, we still have the enum in the keyword field.